crate/server/src/tests/test_utils.rs

```diff
@@ -58,6 +58,8 @@ pub async fn test_app()
             .expect("cannot instantiate KMS server"),
     );
 
+    // --max-run-duration=10m \
+    // --instance-termination-action=DELETE
```

Comment:
just leave a comment to remember to uncomment this

Reply:
actually this is a leftover, deleting...

```suggestion
```